istioctl/main.go

```diff
@@ -21,23 +21,24 @@ import (
 	"io/ioutil"
 	"net/url"
 	"os"
+	"path"
 	"strings"
 	"text/tabwriter"
 
 	"github.com/ghodss/yaml"
-	multierror "github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-multierror"
 	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
```

Comment:
`github.com` and `k8s.io` imports should be separated by a blank line.

Reply:
done